doc/ipython-notebooks/gaussian_process/variational_classifier.ipynb

```diff
@@ -1086,7 +1086,7 @@
     "    inf = inference(kernel_func, features_train, mean_func, labels_train, likelihood, features_inducing)\n",
     "    inf.set_scale(exp(kernel_log_scale))\n",
     "    if optimizing_inducing_points:\n",
-    "        inf.enable_optimizing_inducing_features(True)\n",
+    "        inf.enable_optimizing_inducing_features(True, LBFGSMinimizer())\n",
```

Comment:
@karlnapf 
Users can use `LBFGSMinimizer` instead of `NLOPTMinimizer` to optimize inducing features.